wicket-core/src/main/java/org/apache/wicket/markup/transformer/AbstractTransformerBehavior.java

```diff
@@ -26,17 +31,114 @@
 import org.apache.wicket.request.http.WebResponse;
 
 /**
- * A {@link Behavior} which can be added to any component. It allows to post-process (transform) the
+ * A {@link Behavior} which can be added to any component, allowing to post-process (transform) the
  * markup generated by the component.
- * 
+ * <p>
+ * There's one important limitation with the current implementation: Multiple different instances of
```

Comment:
This Javadoc is obsolete with the improvement below, no ?

Reply:
I disagree, `AbstractTransformerBehavior` itself keeps the limitation and especially existing users should be made aware. The container is only one possible workaround/solution, a different `AbstractTransformerBehavior` itself would be another one. With that the sentence could be removed, but with the container only the sentence still applies.

Reply:
You can just say "If you need to apply several transformations on the same component then use {@linkplain AbstractTransformerBehavior.Multi}".
I see no need to write 20 lines as Javadoc explaining history.

Reply:
Those 20 lines of history are exactly what I regularly miss when running into trouble with Wicket, because without them there's no basic understanding of why things are the way they are. When running into this problem, I needed to look at the entire GIT-history just to find no reason for the current implementation, no clue on how things were supposed to work, for a workaround fitting the existing design etc.

The docs often lack important context from my experience, so I'm trying to add one here.